### PR TITLE
add data frames

### PR DESCRIPTION
**Pull request by zaidalyafeai**

### CODE REVIEW

File: tnkeeh/tnkeeh.py [hunk 1]
```diff
@@ -124,11 +125,85 @@ def _remove_long_words(text, threshold=15):
...
+        if remove_long_words:
+            print('Remove long words')
+            text = _remove_long_words(text)
+
+        text = _add_spaces_to_all_special_chars(text)
+        text = _remove_extra_spaces(text)
+        cleaned_list.append(text)
+    return cleaned_list
+
+def clean_data_frame(df, column_name, segment = False, remove_special_chars = False,
```
Comment (zaidalyafeai) [inline tnkeeh/tnkeeh.py:169]: How to do that more elegantly ? repeating parameters should be done in a better way .

Comment (zaidalyafeai) [reply tnkeeh/tnkeeh.py:169]: @MagedSaeed

Comment (MagedSaeed) [reply tnkeeh/tnkeeh.py:169]: I think we can pass them as a default keyword arguments dictionary. Something like:
```
default_kwargs = dict(
    segment = False,
    remove_special_chars = False, 
    remove_english = False,
    normalize = False,
    remove_diacritics = False,
    execluded_chars = [],
    remove_tatweel = False,
    remove_html_elements = False,
    remove_links = False,
    remove_twitter_meta = False,
    remove_long_words = False,
    remove_repeated_chars = False
)
```
Then, in a function call:
`clean_something(something, **default_kwargs)`

Does this sound fine to you?

Comment (zaidalyafeai) [reply tnkeeh/tnkeeh.py:169]: Yes, will give it a try.

Comment (zaidalyafeai) [reply tnkeeh/tnkeeh.py:169]: Seems to work fine. I refactored some of the code.

**Pull request merged**